src/issuance.rs

```diff
@@ -194,6 +194,11 @@ pub trait IssueAuth: fmt::Debug + Clone {}
 #[derive(Debug, Clone)]
 pub struct Unauthorized;
 
+/// Marker for an unauthorized bundle with correct rho values for each issue note and with no
+/// proofs or signatures.
+#[derive(Debug, Clone)]
+pub struct PartiallyPrepared;
+
```

Comment:
After much consideration, let's go with:
```rust 
/// Marker for an unsigned bundle with no nullifier and no sighash injected.
#[derive(Debug, Clone)]
pub struct AwaitingNullifier;

/// Marker for an unsigned bundle with a Nullifier injected.
#[derive(Debug, Clone)]
pub struct AwaitingSighash;

/// Marker for an unsigned bundle with both Sighash and Nullifier injected.
#[derive(Debug, Clone)]
pub struct Prepared {
    sighash: [u8; 32],
}

/// Marker for an authorized bundle.
#[derive(Debug, Clone, PartialEq, Eq)]
pub struct Signed {
    signature: schnorr::Signature,
}
```
Please make sure we update the comments as well.
@alexeykoren for ref.

Reply:
Also, in tests:
`partially_prepared_bundle` -> `awaiting_sighash_bundle` and similar.

Reply:
Done